.gitignore

```diff
@@ -1,2 +1,5 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+npm-debug.log
+Thumbs.db
```

Comment:
Might be so, but with reference to:
https://github.com/Microsoft/vscode/blob/master/.gitignore